### DISCUSSION TITLE
Cannot create or clone ProjectV2 objects

### DISCUSSION THREAD

**Discussion by heaths** [community/community#18471] [Projects and Issues]
Our team is switching from a third-party solution to Projects V2 and, as part of the [heaths/gh-projects](https://github.com/heaths/gh-projects) GitHub CLI extension, I'd like to enable easily cloning projects including fields and views; however, there are no GraphQL APIs to do so, nor any UI on github.com that would enable this for graphical users.

In general, it'd be great to see more parity with the existing Projects APIs.

**Comment by MathiasReker** [comment]
Those features are coming soon. :-)
